data.txt

```diff
@@ -0,0 +1 @@
+{"platform": [{"Machine": "AMD64", "Processor": "Intel64 Family 6 Model 78 Stepping 3, GenuineIntel", "System OS": "Windows", "Version": "10.0.17134", "CPU Number": "4", "Node": "DESKTOP-B84VDMT", "Memory info[RAM]": "8377131008"}]}
```

Comment:
Add this file to .gitignore. This file generated in runtime, so it should not be in git.